server/startup/migrations/v140.js

```diff
@@ -0,0 +1,17 @@
+import { Migrations } from '../../../app/migrations/server';
+
+import { Messages, Rooms } from '../../../app/models/server';
+
+Migrations.add({
+	version: 140,
+	up() {
+		Messages.find({ drid: { $exists: 1 } }, { fields: { drid: 1 } }).forEach(({ _id, drid }) => Rooms.findOne({ _id: drid }) || Messages.update({ _id }, {
```

Comment:
```suggestion
		Messages.find({ drid: { $exists: 1 } }, { fields: { drid: 1 } }).forEach(({ _id, drid }) => !Rooms.findOne({ _id: drid }) || Messages.update({ _id }, {
```

if I get this right, you're trying to see if the room of the discussion was already removed right? in this case, my suggestion applies